input_examples/topas_parser_test.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
why do use TOPAS specific JSON ? it should accept the same JSON as now SHIELD-HIT12A

Reply:
I thought it will be better for tests to use seperate JSONs, since there could be some differences in parameters used by different simulators. I can change it to just use shieldhit JSON though

Reply:
All simulators accept kinetic energy of primaries. Therefore it should be taken from original JSON, as generated in the UI now.

Reply:
ok